### ISSUE TITLE
Terminology for team support of the W3C Council

### ISSUE THREAD

**Issue by jeffjaffe** [w3c/w3process#588]
The team plays several roles in supporting a W3C Council.  So far, individual documents have kept those roles separate, but we should establish clear distinction in the roles and clear, consistent terminology.

For example, in a draft document which describes the processing of formal objections (https://www.w3.org/2021/05/W3C_Council_Guide.html), the term "assignee" is used to describe the person who tries to resolve the objection and then report to the Council.  

In issue #278, in describing the role of a team person who collects potential reasons for recusal, the term "team shepherd" is used (https://github.com/w3c/w3process/issues/278#issuecomment-964403752).

There is also the role of a team contact for the Council, the person who administratively supports the Council.  (In my mind, the team contact is the same person as the team shepherd, but others might have a different pov.)

**Comment by frivoal** [comment]
> In my mind, the team contact is the same person as the team shepherd, but others might have a different pov

I agree, and think we can keep calling that person the Team contact. No need for new words there.

> the term "assignee" is used to describe the person who tries to resolve the objection and then report to the Council

From the [point of view of the draft director-free process](https://www.w3.org/Consortium/Process/Drafts/director-free/#team-fo-mediation), this is a collective responsibility of the team, which could be, but doesn't have to be, undertaken by a single person.

So far, the process hasn't felt a need for that person to be named, so I'm kind of neutral on that, but I agree that if we keeping having to refer to this role, it'd be good to established some terminology for it. "Assignee" is OK, but a bit too general maye, and possibly in need of a qualifier? "FO Team assignee".

Possible alternatives: mediator, investigator

**Comment by plehegar** [comment]
Current team expectation would be for the assignee to be, by default, the team contact of the Working Group. This is the team person who is the closer to the disputed materials/topics so the person most likely to understand the objection. The analysis provided by the team person would be reviewed by the entire W3C team (similar to Member submission comments). This does add a delay in the timeline since we would need to give time for the entire team to review the analysis and address feedback. My hope is that, overtime, this step can be done in parallel to circulate the analysis to the various parties.

+1 to use "investigator".

**Comment by dwsinger** [comment]
my apologies for the casual use of a new term, shepherd. contact is fine.  investigator is also fine

**Comment by cwilso** [comment]
Actually, I think I may be the one who introduced the term "shepherd", so you can blame me.  Investigator is a fine term.  

I would, however, challenge the assumption that the Team Contact is the default investigator.  Although their expertise may be unparalleled, they are most likely (since they're the Team Contact) to be biased against a formal objection.  I'm not sure that's a great balance.

**Comment by jeffjaffe** [comment]
I think that Chris raises a fair point.  For that reason, and to avoid confusion, it is probably best to have a glossary of each potential team role as a separate term.  If for a certain FO a team member plays more than one role, we can just identify it as such.

**Comment by cwilso** [comment]
Please note that the current language is "assignee".  I think that's okay, compared to "investigator", though it doesn't really capture what their role is supposed to be - I suggested "FO manager" since this role is like a program manager's role, but I think @plehegar didn't like the implications of that. :)

The above-referenced PR changes the language such that I'd be less concerned about the TC being the default for the team's assignment.  PTAL.

**Comment by plehegar** [comment]
I believe there are currently 3 roles in the Team related more or less to the Council:
1. Team Contact (TC): the team person assigned to be a Team Contact for a Working or Interest Group.
2. Team Assignee (TA): the team person responsible for attempting to resolve a Formal Objection and produces a report for the Council if no consensus was found.
3. Council Team Contact (CTC): the team person assigned to manage the FO Council process.

**Comment by frivoal** [comment]
The [current draft of the process](https://www.w3.org/Consortium/Process/Drafts/director-free/) does not distinguish, and merely refers to the Team. However, it may be useful to distinguish in the Guide, and the above terminology from PLH seems fine.

**Comment by frivoal** [comment]
Agreed to close in the 2022/10/28 Process-CG meeting.

**Comment by frivoal** [comment]
Post facto note: "Team Assignee" remains out of scope for here, and in scope for the Guide. "Council Team Contact", on the other hand, has been included in the the Process.

**Comment by frivoal** [comment]
@jeffjaffe as you opened the issue and we are wrapping up this cycle, I'd like to confirm: are you OK with this issue being closed as described in the above comment?
